ServiceDesk/ServiceDeskClient.cs

```diff
@@ -1,20 +1,21 @@
-﻿namespace ServiceDesk
+﻿namespace ArcadiaTeamsBot.ServiceDesk
 {
-    using ServiceDesk.Abstractions;
-    using ServiceDesk.Abstractions.DTOs;
     using System.Collections.Generic;
     using System.Net.Http;
     using System.Threading.Tasks;
     using System.Text.Json;
     using System.Threading;
 
+    using ArcadiaTeamsBot.ServiceDesk.Abstractions;
+    using ArcadiaTeamsBot.ServiceDesk.Abstractions.DTOs;
+
     public class ServiceDeskClient : IServiceDeskClient
     {
-        private const string requestTypesUrl = "https://arcadia-servicedesk-dev31.azurewebsites.net/api/intra/requestTypes";
-
-        private const string currentRequestsUri = "https://arcadia-servicedesk-dev31.azurewebsites.net/api/intra/requests?username=";
-
-        private readonly ServiceDeskRequestPriorityDTO[] priorities = new[] {
+        private const string requestTypesUrl = "intra/requestTypes";
+        private const string currentRequestsUrl = "intra/requests?username=";
+        private readonly IHttpClientFactory clientFactory;
```

Comment:
Please, add empty line between constants and fields.

Reply:
Fixed